diplomacy/web/amplify/backend/function/diplomacyauthconsent/diplomacyauthconsent-cloudformation-template.json

```diff
@@ -0,0 +1,258 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
```

Comment:
We used it to Amplify to:
- Generate the aws auth and backend resources (infra as code) with cloudformation, which in turn was just generated by amplify (no need to author cloudformation or cdk code by hand)
 - This includes cognito, user pool, consent/ rest api endpoint and function that handles the consent/ request
 - Generate all the React Authentication UI scenarios to handle signUp/LogIn/LogOut/ForgotPassword flows.

Reply:
For the most part we may ignore the files under `amplify/ and let the amplify cli tool handle it by itself.